### ISSUE TITLE
Doctor cards must be present on different page

### ISSUE THREAD

**Issue by mohit200008** [mohit200008/medi-Care#35]
The doctor cards Must be present on a separate page And Each doctor will be listed there and the user will be able to filter based on the specialization of the doctor.

**Comment by zugzwang03** [comment]
Hey Mohit sir, I would like to work on this issue under JWOC.

**Comment by mohit200008** [comment]
Go ahead @zugzwang03

**Comment by zugzwang03** [comment]
Hey Mohit sir, I solved this issue under JWOC, but the respective PR count is not allotted to me there. Maybe you missed it...
I successfully completed my pull request also which has been closed.

**Comment by mohit200008** [comment]
wait i will check

**Comment by zugzwang03** [comment]
Yes sir, you need to label my pull request as jwoc and medium, then only the PR will get counted.

**Comment by zugzwang03** [comment]
https://github.com/mohit200008/medi-Care/pull/51 is the link to my closed pull request.

**Comment by mohit200008** [comment]
I changed the labels

**Comment by zugzwang03** [comment]
Sir, you have to label my closed pull request, not the issue.
https://github.com/mohit200008/medi-Care/pull/51 is the link to my closed pull request. 
The JWOC authorities are referring to the labels of the closed pull requests for PR count.

**Comment by mohit200008** [comment]
Please check now

**Comment by zugzwang03** [comment]
Yeah, it's done. Thank you sir :)

**Comment by Han9128** [comment]
Hi mohit i want to take this issue may you assign it to me?

**Comment by mohit200008** [comment]
okay go ahead @Han9128
